components/blog/blog-list.tsx

```
@@ -136,7 +102,10 @@ export function BlogList({ category }: { category?: string }) {
                       )}
                       <div className={post.image ? "md:w-2/3" : "w-full"}>
                         <CardHeader>
-                          <CardTitle className="text-xl">{post.title}</CardTitle>
+                          <CardTitle
+                            className="text-xl"
+                            dangerouslySetInnerHTML={{ __html: post.title }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)